### ISSUE TITLE
Missing tooltip for the `...` 3-dots/ellipsis menu icon

### ISSUE THREAD

**Issue by stephendonner** [brave/brave-browser#33732]
## Description 

Missing tooltip for the `...` 3-dots/ellipsis menu icon

## Steps to Reproduce

1. install 1.61.33`
2. launch Brave
3. restart (to pick up Griffin seed)
4. load an article
5. click on `Brave Leo`
6. click on `Accept and begin`
7. hover over the top-right `...` ellipsis/3-dots menu icon

## Actual result:

No tooltip

<img width="394" alt="Screen Shot 2023-10-18 at 2 28 02 PM" src="https://github.com/brave/brave-browser/assets/387249/894d5b80-0e00-4a91-a8c0-dcc8bc7cde81">

## Expected result:

Unsure what it should say, precisely, but we should have a tooltip for this

## Reproduces how often: 

100%

## Brave version (brave://version info)

```
 Brave | 1.61.33 Chromium: 118.0.5993.88 (Official Build) nightly (x86_64)
-- | --
Revision | 84e29e8281b718324e93d3bc23a9954e0620063e
OS | macOS Version 11.7.10 (Build 20G1427)
```

## Version/Channel Information:

- Can you reproduce this issue with the current release? 
- Can you reproduce this issue with the beta channel? 
- Can you reproduce this issue with the nightly channel? 

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards? 
- Is the issue reproducible on the latest version of Chrome? 

## Miscellaneous Information:

cc @bbondy @mattmcalister @aguscruiz @petemill @nullhook @brave/qa-team

**Comment by aguscruiz** [comment]
Should probably say "Leo settings"

**Comment by kjozwiak** [comment]
The above requires `1.60.109` or higher for `1.60.x` verification 👍

**Comment by GeetaSarvadnya** [comment]
Verification **PASSED** on

```
Brave | 1.60.109 Chromium: 119.0.6045.59 (Official Build) (64-bit)
-- | --
Revision | fc399c7c433510eb03c2b35281bf8935804b99de
OS | Windows 10 Version 22H2 (Build 19045.3570)

```
- Confirmed that the tooltip is shown for the ... 3-dots/ellipsis menu icon

Example | Example
----------|------------
![image](https://github.com/brave/brave-browser/assets/38657976/2f7bca7d-6cda-4859-9e4c-673b99b396c0) | ![image](https://github.com/brave/brave-browser/assets/38657976/e853d403-1aeb-4ec7-bcd7-db1b4894bace)
